platform/core/PolyPodCoreAndroid/core/src/main/kotlin/coop/polypoly/core/JniApi.kt

```diff
@@ -37,7 +37,7 @@ object JniApi {
         fsRoot: String,
         callback: JniApi
     ): ByteArray
-    external fun loadFeatureCategories(featuresDir: String): ByteArray
+    external fun loadFeatureCategories(args: ByteArray): ByteArray
```

Comment:
The idea is, apparently, to move away from encoding/decoding individual parameters, and just passing all the arguments for every API call in message pack encoded. So going with the flow here.